Assignment4/JavierSabin-190299/task07.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+"""Task07.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1tV5j-DRcpPtoJGoMj8v2DSqR_9wyXeiE
+
+**Task 07: Querying RDF(s)**
+"""
+
+!pip install rdflib 
+github_storage = "https://raw.githubusercontent.com/FacultadInformatica-LinkedData/Curso2020-2021/master/Assignment4"
+
+"""Leemos el fichero RDF de la forma que lo hemos venido haciendo"""
+
+from rdflib import Graph, Namespace, Literal
+from rdflib.namespace import RDF, RDFS
+g = Graph()
+g.namespace_manager.bind('ns', Namespace("http://somewhere#"), override=False)
+g.namespace_manager.bind('vcard', Namespace("http://www.w3.org/2001/vcard-rdf/3.0#"), override=False)
+g.parse(github_storage+"/resources/example6.rdf", format="xml")
+
+"""TASK 7.1: List all subclasses of "Person" with RDFLib and SPARQL"""
+# RDF
+ns = Namespace("http://somewhere#")
+for s,p,o in g.triples((None, RDFS.subClassOf, ns.Person)):
+    print(s)
+for s2,p2,o2 in g.triples((None, RDFS.subClassOf, s)):
+    print(s2)
```

Comment:
this is not a well formed loop for what you are intending.
For the purpose of the exercise, you only need the first one